src/DependencyInjection/Configuration.php

```diff
@@ -39,27 +39,6 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->end()
                     ->end()
                 ->end()
-                ->arrayNode('openid_provider_options')
-                    ->isRequired()
-                    ->children()
-                        ->scalarNode('configuration_url')
-                            ->info('URL to OpenId Discovery Document')
-                            ->isRequired()
-                        ->end()
-                        ->scalarNode('client_id')
-                            ->info('Client ID assigned by authorizer')
-                            ->isRequired()->cannotBeEmpty()
-                        ->end()
-                        ->scalarNode('client_secret')
-                            ->info('Client secret/password assigned by authorizer')
-                            ->isRequired()->cannotBeEmpty()
-                        ->end()
-                        ->scalarNode('callback_uri')
-                            ->info('Callback URI registered at identity provider')
-                            ->isRequired()->cannotBeEmpty()
-                        ->end()
-                    ->end()
-                ->end()
             ->end();
```

Comment:
Why change this completely? I would think it better to expand this to allow for an array of `openid_provider_options` and then have a factory generate a service for each. 

This would allow the config for the bundle to remain in the bundles package yaml, and not push bundle config into the using systems services. Doing this breaks Symfony config conventions, and makes tools like `bin/console config:dump-reference` and `bin/console debug:config` less useful.